res/sections/50-Metrics.tex

```diff
@@ -0,0 +1,91 @@
+\section{Results}
+
+During GameBank development we have created some metrics in order to gain 
+additional information about our application behaviour. In particular, we 
+have focused our attention on Bluetooth, creating a counter of the data 
+received and transmitted for a particular device. Only the payload transmitted 
+has been counted tough, since we weren't able to intercept data communications 
+between devices that would have provided better measurements. The metrics are 
+stored in a file in a \texttt{csv} format, containing the type of transmission 
+(input or output), the quantity of the data exchanged and the type of the 
+event in the \texttt{BTBundle}. To ease any doubt about the terminology, we 
+defined output data as the data a device transmits to another one (outgoing 
+transmission), while for input data we intend the data a device receive 
+(incoming transmission).
+
+\subsection{Devices}
+
+\begin{table}[t]
+ \centering
+ \caption{Table of the devices used to test GameBank}
+ \label{tab:res:lod}
+ \begin{tabular}{l r}
+  \textbf{Device} & \textbf{Android Version} \\ \toprule
+  ASUS ZenFone Go & 6.0.1 \\
+  WIKO U Feel Lite & 6.0.0 \\
+  SAMSUNG S3 & 7.1.0 \\
+  HUAWEI P9 & 7.0.0 \\
+  MOTOROLA MotoG3 & 6.0.1 \\
+  SAMSUNG Nexus 10 & 5.1.1 \\
+  XIAOMI RedMi 4 Pro & 6.0.1
+ \end{tabular}
+\end{table}
+
+In order to test our application we used specific devices listed in 
```

Comment:
Perche' hai usato il termine 'specific'?

Reply:
Perché abbiamo usato un determinato elenco di dispositivi, che appunto abbiamo messo in tabella! :blush: